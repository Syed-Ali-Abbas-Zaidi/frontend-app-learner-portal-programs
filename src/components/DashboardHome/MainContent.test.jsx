import React from 'react';
import renderer from 'react-test-renderer';

import MainContent from './MainContent';

describe('<MainContent />', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create((
        <MainContent />
      ))
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
