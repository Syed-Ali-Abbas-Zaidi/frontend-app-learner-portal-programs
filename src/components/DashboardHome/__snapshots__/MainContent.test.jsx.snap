// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MainContent /> renders correctly 1`] = `
<div
  className="d-flex justify-content-center align-items-center"
>
  <div
    className="spinner-border text-primary"
    role="status"
  >
    <div
      className="sr-only"
    >
      Loading program enrollments...
    </div>
  </div>
</div>
`;
